src/components/BlogCard/index.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../style/variables";
+
+.blogPost {
+  height: 20rem;
+  border-radius: 1rem !important;
+  box-shadow: 0px 0px 10px 0 rgba(0, 0, 0, 0.06);
+  margin-bottom: 2rem;
+
+  img {
+    border-top-left-radius: calc(1rem - 1px) !important;
+    border-top-right-radius: calc(1rem - 1px) !important;
+    object-fit: cover !important;
+    object-position: 0 25% !important;
+    height: 15rem;
+  }
+
+  .blogPostTitle {
+    text-align: center;
+    color: $primaryColor;
+  }
+
+  &:hover {
+    text-decoration: underline;
+    color: $primaryColor;
+  }
+
+  a {
+    color: $primaryColor;
```

Comment:
this one is repeated three times. I suggest to refactor it, give it to .blogPost for example and set this to `inherit` or try smth else

Reply:
ok done it actually works without it